lib/blocs/tasks_bloc.dart

```diff
@@ -58,6 +58,10 @@ class _TasksBloc {
           .replaceAll('{np}', ' ### $negativePrompt');
       print(formattedPrompt);
 
+      var isDonateImageEnabled = await sharedPrefsBloc.isDonateImageEnabled();
+      final shouldShare =
+          isDonateImageEnabled && img2ImgInputEncodedString == null;
```

Comment:
@ndahlquist Is there anything else I'm missing?

Reply:
Nope looks good!